ui/imports/shared/views/HistoryView.qml

```diff
@@ -190,19 +192,19 @@ ColumnLayout {
         TransactionDelegate {
             width: ListView.view.width
             modelData: model
-            isIncoming: isModelDataValid ? modelData.to === root.overview.mixedcaseAddress: false
+            transactionType: isModelDataValid && modelData.to.toLowerCase() === root.overview.mixedcaseAddress.toLowerCase() ? TransactionDelegate.Receive : TransactionDelegate.Send
             currentCurrency: RootStore.currentCurrency
             cryptoValue: isModelDataValid ? modelData.value.amount : 0.0
             fiatValue: isModelDataValid ? RootStore.getFiatValue(cryptoValue, symbol, currentCurrency): 0.0
             networkIcon: isModelDataValid ? RootStore.getNetworkIcon(modelData.chainId) : ""
             networkColor: isModelDataValid ? RootStore.getNetworkColor(modelData.chainId) : ""
-            networkName: isModelDataValid ? RootStore.getNetworkShortName(modelData.chainId) : ""
+            networkName: isModelDataValid ? RootStore.getNetworkFullName(modelData.chainId) : ""
             symbol: isModelDataValid && !!modelData.symbol ? modelData.symbol : ""
             transferStatus: isModelDataValid ? RootStore.hex2Dec(modelData.txStatus) : ""
-            shortTimeStamp: isModelDataValid ? LocaleUtils.formatTime(modelData.timestamp * 1000, Locale.ShortFormat) : ""
-            savedAddressNameTo: isModelDataValid ? RootStore.getNameForSavedWalletAddress(modelData.to) : ""
-            savedAddressNameFrom: isModelDataValid ? RootStore.getNameForSavedWalletAddress(modelData.from) : ""
-            isSummary: true
+            timeStampText: isModelDataValid ? LocaleUtils.formatRelativeTimestamp(modelData.timestamp * 1000) : ""
+            savedAddressNameTo: isModelDataValid ? WalletStores.RootStore.getNameForWalletAddress(modelData.to) : ""
```

Comment:
Saved addresses are different than wallet account. You will need to find this in the list of saved addresses, not wallet address.

Maybe it is suppose to support both wallet address + save addresses?

Reply:
@alaibe Thanks for pointing that out! I am still not fluent with all app terminology. 
Idea is to retrieve name for address from saved addresses and then try wallet names, so that I can see my wallet name when address is used in any text.

Changed name to get general name instead of specific one.